src/Missings.jl

```diff
@@ -1,7 +1,7 @@
 module Missings
 
 export allowmissing, disallowmissing, ismissing, missing, missings,
-       Missing, MissingException, levels, coalesce
+       Missing, MissingException, levels, coalesce, passmissing
```

Comment:
I personally prefer `Missings.propagate` if we're still bikeshedding the name here.

Reply:
I like `propagate` too, the only gripe is that it's quite long. But well...

Reply:
But do we want to export it or not? If yes then `propagate` is ok, but `Missings.propagate` is a bit long.

Reply:
Unfortunately `propagate` is quite general to claim it for this feature...

Reply:
I know, but that is why I thought the idea was `propagatemissing` which tab-completes.